msmbuilder/libdistance/libdistance.pyx

```diff
@@ -126,6 +131,56 @@ def assign_nearest(X, Y, const char* metric, npy_intp[::1] X_indices=None):
         raise TypeError('X and y must be both float32 or float64')
 
 
+def cdist(XA, XB, const char* metric):
+    """cdist(XA, XB, metric):
+
+    Computes distance between each pair of the two collections of inputs.
+
+    Parameters
+    ----------
+    XA : array, shape = (n_samples, n_features) or md.Trajectory
+        The data array
+    XB : array, shape = (n_samples, n_features) or md.Trajectory
```

Comment:
The `shape` documentation here should use a different variable for the length of XA and the length of XB, since they don't need to have the same length (but do need to have the same n_features).